server/zanata-war/src/test/java/org/zanata/adapter/GettextAdapterTest.java

```diff
@@ -105,5 +111,48 @@ public void testGettextReference() {
         assertThat(potEntryHeader.getReferences().get(0)).isEqualTo("reference");
     }
 
+    @Test
+    public void testTranslatedGettext() {
+
+        TranslationsResource transResource = new TranslationsResource();
+        addTranslation(transResource, "0293301ed6a54b7e4503e74bba17bf11", "Carpeta padre", ContentState.Approved);
+        addTranslation(transResource, "47a0be8d1015d526a1fbaa56c3102135", "Asunto:", ContentState.Translated);
+        addTranslation(transResource, "49ab28040dfa07f53544970c6d147e1e", "Conectar", ContentState.NeedReview);
```

Comment:
As I said with @djansen-redhat's TS test, I don't think we should hard-code hashes like these unless we're unit testing a hashing method. It should be possible to avoid this by parsing a PO/POT file first.

Also, it seems like these translations already exist in the PO file you're parsing below, so the test might pass even if addTranslation did nothing. I suggest removing these translations from the PO file (and then call it a POT file). 

Unless you want to keep a copy of the full PO file, to create another test for the parsing of translations, which would be a good idea.